classes/class-wc-connect-shipping-label.php

```diff
@@ -473,16 +473,11 @@ public function get_label_payload( $post_order_or_id ) {
 		public function meta_box( $post ) {
 			$order = wc_get_order( $post );
 
-			$order_id = WC_Connect_Compatibility::instance()->get_order_id( $order );
-
 			$payload = $this->get_label_payload( $order );
 			if ( ! $payload ) {
 				return;
 			}
 
-			//pass order ID as the only constant in the context, everything else can be fetched dynamically
-			$payload[ 'orderId' ] = $order_id;
```

Comment:
It looks like this change broke the async load of label data - can the order id be retrieved elsewhere in the client code?